integrations/tensorflow/e2e/BUILD

```diff
@@ -12,6 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Test coverage across backends for e2e tests is defined directly in the BUILD
+# files. A coverage table generated from this file can be viewed here:
+#   https://google.github.io/iree/TensorFlowE2ECoverage
+# Updates made to test suite names should also be reflected here:
+#   https://github.com/google/iree/blob/main/scripts/update_e2e_coverage.py
```

Comment:
```suggestion
#   https://github.com/google/iree/blob/master/scripts/update_e2e_coverage.py
```

nit: keep using `master` for consistency for now. A change to Copybara will go through and update all refs together when finish shuffling branches around. (Ugh, hoping there are less silly things like this to spot once this is over :P)